src/components/Chart/RoundsChart.js

```diff
@@ -0,0 +1,162 @@
+import cx from 'classnames';
+import * as d3 from 'd3';
+import { groupBy } from 'lodash';
+import React, { useEffect, useRef, useState } from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+import Axis from './Axis';
+import Line from './Line';
+
+import { getPlayerInfo } from '../../data/utils';
+
+import styles from './styles/RoundsChart.module.scss';
+
+const useResizeObserver = (
+  containerRef,
+  initialSize = { height: 0, width: 0 }
+) => {
+  const [size, setSize] = useState(initialSize);
+
+  useEffect(() => {
+    const containerElem = containerRef.current;
+
+    if (!containerElem) {
+      return;
+    }
+
+    const observer = new ResizeObserver(() => {
+      setSize({
+        height: containerElem.clientHeight,
+        width: containerElem.clientWidth,
+      });
+    });
+
+    observer.observe(containerElem);
+
+    return () => observer.unobserve(containerElem);
+  }, [containerRef]);
+
+  return size;
+};
+
+const translate = (x, y) => `translate(${x}, ${y})`;
+
+const COLORS = [
```

Comment:
If we have to add a player, do we have to add a color to this list? Not sure of how we can make that more obvious, if so. While it feels icky, players could have colors assigned in the PLAYERS constant. Or there could be code that automatically pics more colors.

Reply:
This is a static subset of colors from a d3 color scale. We can do it in a more dynamic way, this was more to get it working.